spec/automation/unit/method_validation/amazon_pre_retirement_spec.rb

```diff
@@ -20,12 +20,12 @@
   it "calls stop for ebs instances" do
     @vm.hardware = @ebs_hardware
     MiqAeEngine.instantiate("#{@ins}?Vm::vm=#{@vm.id}#amazon", @user)
-    MiqQueue.exists?(:method_name => 'stop', :instance_id => @vm.id, :role => 'ems_operations').should be_true
+    expect(MiqQueue.exists?(:method_name => 'stop', :instance_id => @vm.id, :role => 'ems_operations')).to be_truthy
   end
 
   it "should not call stop for instance store instances" do
     @vm.hardware = @is_hardware
     MiqAeEngine.instantiate("#{@ins}?Vm::vm=#{@vm.id}#amazon", @user)
-    MiqQueue.exists?(:method_name => 'stop', :instance_id => @vm.id, :role => 'ems_operations').should be_false
+    expect(MiqQueue.exists?(:method_name => 'stop', :instance_id => @vm.id, :role => 'ems_operations')).to be_falsey
```

Comment:
these are either true/false, not sure if we should be using truthy/falsey... not a problem, just not sure when we should be using them vs. be_true/be_false